int16 a0
int16 a1
int16 a2
int16 a3
int16 a4
int16 aGrip